contrib/src/main/java/com/twelvemonkeys/contrib/tiff/TIFFUtilities.java

```diff
@@ -517,6 +664,7 @@ public void rotate(int degree) {
     /**
      * TODO: Temporary clone, to be removed after TMI204 has been closed
      */
+
```

Comment:
Yes, I'm not sure exactly how yet, but I addressed this in the review summary above. The intent here was just to remove the reference to #204. We might need a new issue though. :-P